src/dense/rosenbrock_interpolants.jl

```diff
@@ -100,3 +100,27 @@ end
   @views @.. out = Θ1*y₀[idxs] + Θ*(y₁[idxs] + Θ1*(k[1][idxs] + Θ*k[2][idxs]))
   out
 end
+
+# First Derivative 
+
+@muladd function _ode_interpolant(Θ,dt,y₀,y₁,k,cache::Rodas4ConstantCache,idxs::Nothing,T::Type{Val{1}})
+  @inbounds (k[1] - y₀ + y₁ - 2*k[1]*Θ + 2*k[2]*Θ-3*k[2]*Θ^2)/dt
```

Comment:
Is it possible to rearrange this with [Horner's method](https://en.wikipedia.org/wiki/Horner%27s_method), as in the interpolations above?

Reply:
And in the other implementations below, of course.

Reply:
done.